docker/pypi/wmagent-mariadb/manage

```diff
@@ -101,12 +101,12 @@ version(){
 
 
 init_mariadb(){
-    # The function to set/check initial database configurations and user preveleges
```

Comment:
Why did you remove this check and substitute it with the one from above. They do check two different things. 
* The first one checks if the current user is the admin for the database
* The second one checks if somebody has configured one and the same admin user to serve as  admin and databse user.
 
We've have had this conversation in the past. It was long. It was decided to stay with the proper separation between users to drive the database with root privileges and a user to be connecting from outside as a user of  just for the database. Please, lets not mess up with this part, in this commits and changes.